packages/documentation/src/stories/get-started/styles.stories.mdx

```diff
@@ -0,0 +1,58 @@
+import { Meta } from '@storybook/addon-docs';
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+
+<Meta
+  title="Get Started/Styles"
+  parameters={{
+    previewTabs: {
+      canvas: {
+        hidden: true,
+      },
+    },
+    viewMode: 'docs',
+    badges: [BADGE.NEEDS_REVISION],
+  }}
+/>
+
+# @swisspost/design-system-styles
+
+[![npm version](https://badge.fury.io/js/@swisspost%2Fdesign-system-styles.svg)](https://badge.fury.io/js/@swisspost%2Fdesign-system-styles)
+
+## Installation
+
+```bash dark
+  npm install @swisspost/design-system-styles
+```
+
+## Usage
+
+Import one of our stylesheets into your project (for example into your `/src/styles.scss` file).
+
+```scss dark
+@use '@swisspost/design-system-styles/<bundle-name>.<extension>';
+```
+
+### Available bundles
+
+| Bundle name | Bundle extensions | Description                                                        |
+| :---------- | :---------------- | :----------------------------------------------------------------- |
+| `index`     | `css`, `scss`     | everything for internet facing applications                        |
+| `intranet`  | `css`, `scss`     | everything for internal applications                               |
+| `basics`    | `css`, `scss`     | atomic styles for font, buttons, lists, etc. (no component styles) |
```

Comment:
```suggestion
| `basics`    | `css`, `scss`     | atomic styles for font, buttons, lists, etc. (no component styles) |
| `core`      | `scss`              | variables, mixins, functions and placeholders (no CSS output) |
```

Reply:
The core is only used to write custom styles and therefore no entry file.
That's why I described the usage of it in its own subsection.